swagger.yaml

```diff
@@ -285,561 +786,616 @@ paths:
-        type: "number"
+        type: number
       position_in_class:
-        type: "number"
+        type: number
       gap_to_p1_in_class:
-        description: "in seconds"
-        type: "number"
-        format: "double"
+        description: in seconds
+        type: number
+        format: double
       start_position_in_class:
-        type: "number"
+        type: number
       time:
-        description: "in seconds"
-        type: "number"
-        format: "double"
+        description: in seconds
+        type: number
+        format: double
       speed:
-        description: "in unit of event - either km/h or mph"
-        type: "number"
-        format: "double"
-  
+        description: in unit of event - either km/h or mph
+        type: number
+        format: double
   ResultLap:
     description: A lap inside a result which includes run-wide details like average_speed and total_laps
-    allOf: 
+    allOf:
       - $ref: '#/definitions/Lap'
-      - type: "object"
+      - type: object
         properties:
           total_laps:
-            type: "number"
-          total_time: 
-            type: "string"
-            format: "date-time"
+            type: number
+          total_time:
+            type: string
+            format: date-time
           fastest:
-            type: "boolean"
+            type: boolean
           average_speed:
-            description: "in unit of event - either km/h or mph"
-            type: "number"
-            format: "double"
+            description: in unit of event - either km/h or mph
+            type: number
+            format: double
           best_speed:
-            description: "in unit of event - either km/h or mph"
-            type: "number"
-            format: "double"
+            description: in unit of event - either km/h or mph
+            type: number
+            format: double
           status:
-            type: "string"
+            type: string
             enum:
               - active
               - finished
               - pits
               - dns
-              - dsq 
-              - dnf  
-  
+              - dsq
+              - dnf
   RacerLapArray:
-    description: A racer with a laps property that contains an array of laps          
+    description: A racer with a laps property that contains an array of laps
     allOf:
       - $ref: '#/definitions/Racer'
-      - type: "object"
+      - type: object
         properties:
-          laps: 
+          laps:
             description: An ordered array of laps for this racer where the last member of the array is how they finished the session
-            type: "array"
+            type: array
             items:
               $ref: '#/definitions/Lap'
-
   Flag:
-    description: "Control flag"
+    description: Control flag
     properties:
       name:
         description: A simple string describing the flag condition (subject to change)
-        type: "string"
+        type: string
```

Comment:
@chrisredrich this too I think needs to be an enum of the allowed flags we accept (warmup, green, yellow, red, finish, ...?)

Reply:
@tejaswinichile could you update with the same flag options as above?